cmd/template/chiRoutes.go

```diff
@@ -47,5 +49,38 @@ func (s *Server) helloWorldHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write(jsonResp)
 }
 
+func (s *Server) pingPongWebsocketHandler(w http.ResponseWriter, r *http.Request) {
+	socket, err := websocket.Accept(w, r, nil)
+
+	if err != nil {
+		log.Print("could not open websocket")
+		// pray this works for your user
```

Comment:
?? lol

Reply:
I think it's odd to handle error cases in the error handling block so I just ignored the potential error returned in the line under that and put a comment there indicating a potential issue but I can just remove this comment.